apps/discord/auth.py

```
@@ -0,0 +1,42 @@
+"""Authentication and request verification for Discord interactions."""
+
+import os
+from typing import Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)